<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clock</class>
 <widget class="QMainWindow" name="clock">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>776</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btn_start">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>139</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_stop">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>140</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcd_play">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>9</y>
      <width>751</width>
      <height>121</height>
     </rect>
    </property>
    <property name="digitCount">
     <number>12</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_point">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>140</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>打点</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_pause">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>140</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>暂停</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="tb_display">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>751</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>140</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>776</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
